_includes/privacy_policy_export.html

```diff
@@ -0,0 +1,202 @@
+<li style="font-weight: 400;"><span style="font-weight: 400;">demographic information such as postcode, preferences and interests;</span></li>
+<li style="font-weight: 400;"><span style="font-weight: 400;">financial information such as credit / debit card numbers;</span></li>
+</ol>
+</ol>
+<p><span style="font-weight: 400;">in each case, in accordance with this privacy policy.</span></p>
+<p><strong>How we collect Data</strong></p>
+<p><span style="font-weight: 400;">6. We collect Data in the following ways:</span></p>
+<ol>
+<ol style="list-style-type: lower-alpha;">
+<li style="font-weight: 400;"><span style="font-weight: 400;">data is given to us by you ; </span></li>
+<li style="font-weight: 400;"><span style="font-weight: 400;">data is received from other sources; and</span></li>
+<li style="font-weight: 400;"><span style="font-weight: 400;">data is collected automatically.</span></li>
+</ol>
+</ol>
+<p><strong>Data that is given to us by you</strong></p>
+<p><span style="font-weight: 400;">7. SOFTWARE CRAFT AND TESTING UK CIC t/a SoCraTes UK will collect your Data in a number of ways, for example:</span></p>
+<ol>
+<ol style="list-style-type: lower-alpha;">
+<li style="font-weight: 400;"><span style="font-weight: 400;">when you contact us through the Website, by telephone, post, e-mail or through any other means;</span></li>
+<li style="font-weight: 400;"><span style="font-weight: 400;">when you complete surveys that we use for research purposes (although you are not obliged to respond to them);</span></li>
+<li style="font-weight: 400;"><span style="font-weight: 400;">when you enter a competition or promotion through a social media channel;</span></li>
+<li style="font-weight: 400;"><span style="font-weight: 400;">when you make payments to us, through this Website or otherwise;</span></li>
+<li style="font-weight: 400;"><span style="font-weight: 400;">when you elect to receive marketing communications from us;</span></li>
+</ol>
+</ol>
+<p><span style="font-weight: 400;">in each case, in accordance with this privacy policy.</span></p>
+<p><strong>Data that is received from third parties</strong></p>
+<p><span style="font-weight: 400;">8. SOFTWARE CRAFT AND TESTING UK CIC t/a SoCraTes UK will receive Data about you from the following third parties:</span></p>
+<ol>
+<ol style="list-style-type: lower-alpha;">
+<li style="font-weight: 400;"><span style="font-weight: 400;">Eventbrite; </span></li>
+<li style="font-weight: 400;"><span style="font-weight: 400;">Google.</span></li>
+</ol>
+</ol>
+<p><strong>Data that is received from publicly available third parties sources</strong></p>
+<p><span style="font-weight: 400;">9. We will receive Data about you from the following publicly available third party sources:</span></p>
+<ol>
+<ol style="list-style-type: lower-alpha;">
+<li style="font-weight: 400;"><span style="font-weight: 400;">Github; </span></li>
```

Comment:
do we need to be more specific and use proper company names here?

Reply:
I think trading names are sufficient, but it's definitely better to provide full name so will update